README.md

```diff
@@ -78,6 +78,12 @@ We support [OpenTelemetry](https://opentelemetry.io) as the library which you ca
 - Python
 - NodeJS
 - Go
+- PHP
+- .NET
+- Ruby on Rails
```

Comment:
```suggestion
- Ruby
```

Reply:
Lets keep RoR for now. We have docs only for that